ts/features/bonus/cgn/saga/orchestration/eyca/eycaActivationSaga.ts

```diff
@@ -5,11 +5,19 @@ import {
   navigateToCgnDetails,
   navigateToEycaActivationLoading
 } from "../../../navigation/actions";
-import { handleEycaActivationSaga } from "../../networking/eyca/activation/getEycaActivationSaga";
+import {
+  getActivation,
+  handleEycaActivationSaga,
+  handleStartActivation
+} from "../../networking/eyca/activation/getEycaActivationSaga";
 import { navigationHistoryPop } from "../../../../../../store/actions/navigationHistory";
-import { cgnEycaActivationCancel } from "../../../store/actions/eyca/activation";
+import {
+  cgnEycaActivation,
+  cgnEycaActivationCancel
+} from "../../../store/actions/eyca/activation";
 import { BackendCGN } from "../../../api/backendCgn";
 import { cgnEycaStatus } from "../../../store/actions/eyca/details";
+import { SagaCallReturnType } from "../../../../../../types/utils";
 
 export function* eycaActivationWorker(
```

Comment:
This PR lgtm
could you leave the function doc explaining what it does?

Reply:
@Undermaken added within e14ad76